code-challenges.js

```diff
@@ -20,49 +20,92 @@ const fruit2 = "banana"
 
 // --------------------1) Create the code that determines if a given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
-// Describe your process:
+// Describe your process: I want to make a conditional statement
+// in that statement I want it to console.log whatever the temp is below or above boiling point and my catch all is the same"
+
+// const temp = 42
+
+// if(temp > 212){
+//     console.log(temp + " is above boiling point")
+// } else if(temp < 212){
+//     console.log(temp + " is below boiling point")
+// } else(consol.log(temp + " is at boiling point"))
 
-const temp = 42
 // Expected output: "42 is below boiling point"
 
 // const temp = 350
-// Expected output: "350 is above boiling point"
+
+// if(temp > 212){
+//     console.log(temp + " is above boiling point")
+// } else if(temp < 212){
+//     console.log(temp + " is below boiling point")
+// } else(console.log(temp + " is at boiling point"))
+
+// // Expected output: "350 is above boiling point"
+
 
 // const temp = 212
+
+// if(temp > 212){
+//     console.log(temp + " is above boiling point")
+// } else if(temp < 212){
+//     console.log(temp + " is below boiling point")
+// } else(console.log(temp + " is at boiling point"))
+
 // Expected output: "212 is at boiling point"
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
-// Describe your process:
+// Describe your process: I will just console.log both const and put a "+" to add them together but add .length to both in order to add both arrays into one and count the length.
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+
+// console.log(padres1984WorldSeriesRuns.length + padres1998WorldSeriesRuns.length)
+
 // Expected output: 9
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
-// Describe your process:
+// Describe your process: I will need to turn the string into an array by splitting it and then reversing the aray then joining it again
 
 const currentCohort = "Foxtrot 2022"
+
+// const arrayCurrentCohort = currentCohort.split("")
+
+// console.log(arrayCurrentCohort.reverse().join(""))
```

Comment:
Great work on this!  You can chain as many built in methods as you want, which could simplify this a little further.